### PR TITLE
move into a multi-project setup

### PR DESCRIPTION
**Pull request by bavardage**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,50 +15,17 @@ plugins {
...
 apply plugin: 'com.palantir.baseline-config'
 
-sourceCompatibility = 1.7
-
-group 'com.palantir.ri'
-version gitVersion()
-
 repositories {
     jcenter()
 }
```
Comment (markelliot) [inline build.gradle:23]: this should move to `allprojects`

File: build.gradle [hunk 2]
```diff
@@ -70,23 +37,5 @@ tasks.coveralls {
...
-}
-
-tasks.bintrayUpload.onlyIf {
-    System.getenv('TRAVIS_TAG') && project.version ==~ /\d+\.\d+\.\d+/
-}
-
-tasks.bintrayUpload.dependsOn { generatePomFileForBintrayPublication }
-tasks.bintrayUpload.dependsOn { sourceJar }
-tasks.bintrayUpload.dependsOn { build }
+tasks.eclipse.dependsOn(tasks.cleanEclipse)
```
Comment (markelliot) [inline build.gradle]: generally prefer not to add this

File: resource-identifier-utils/build.gradle [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apply from: "${rootDir}/gradle/publish-libs.gradle"
```
Comment (markelliot) [inline resource-identifier-utils/build.gradle]: let's not add resource-identifier-utils as part of this changeset

File: resource-identifier/build.gradle [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apply from: "${rootDir}/gradle/publish-libs.gradle"
+
+dependencies {
+    compile "com.fasterxml.jackson.core:jackson-annotations:$jacksonVersion"
```
Comment (markelliot) [inline resource-identifier/build.gradle]: we should clean up variable use here to use braces (e.g. `${jacksonVersion}`)

**Comment by markelliot** [comment]
nits, but otherwise sane

File: build.gradle (hunk 1 above)
Comment (bavardage) [inline build.gradle:23]: done

File: build.gradle (hunk 2 above)
Comment (bavardage) [inline build.gradle]: think was there already, but removed

File: resource-identifier-utils/build.gradle (hunk 3 above)
Comment (bavardage) [inline resource-identifier-utils/build.gradle]: removed

File: resource-identifier/build.gradle (hunk 4 above)
Comment (bavardage) [inline resource-identifier/build.gradle]: done

**Comment by bavardage** [comment]
so I removed the .travis.yml :S

**Comment by bavardage** [comment]
(and pulled the gradles from ECV)

File: build.gradle [hunk 5]
```diff
@@ -1,92 +1,53 @@
...
+repositories {
+    jcenter() // needed for baselineUpdateConfig task
 }
 
-task sourceJar(type: Jar) {
-    from sourceSets.main.allSource
-    classifier 'sources'
+allprojects {
+    version gitVersion()
+    group 'com.palantir.palantir.ri'
```
Comment (markelliot) [inline build.gradle]: Think this is a typo

Comment (bavardage) [inline build.gradle]: ooops yop fixing

Comment (bavardage) [inline build.gradle]: done

**Comment by uschi2000** [comment]
this is pretty old, closing.

**Pull request closed without merging**